ProcessList.c

```diff
@@ -494,6 +494,15 @@ void ProcessList_expandTree(ProcessList* this) {
    }
 }
 
+void ProcessList_collapseAllBranches(ProcessList* this) {
+   int size = Vector_size(this->processes);
+   for (int i = 0; i < size; i++) {
+      Process* process = (Process*) Vector_get(this->processes, i);
+      // FreeBSD has pid 0 = kernel and pid 1 = init, so init has tree_depth = 1
+      if (process->tree_depth > 0 && process->pid > 1) process->showChildren = false;
```

Comment:
Please line-wrap the body of that if statement.

Reply:
done